### PR TITLE
Added Discord BotList

### PR DESCRIPTION
**Pull request by OGMatrix**
**View on Main Page:**
ss1

**View on Profile Page:**
ss4

**View on Bot Page:**
ss3

**View on Vote Page:**
ss2

**There are 8 more sites but this would be too Long!**

### CODE REVIEW

**Comment by StrikerFRFX** [comment]
that latest commit just makes me wanna ask more kekw.

**Comment by SuperzZyntexYT** [comment]
> that latest commit just makes me wanna ask more kekw.

Just removed a semicolon or two lmao

File: websites/D/Discord BotList/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "color": "#3563ed",
+    "tags": [
+      "discord",
+      "botlist",
+      "list"
+    ],
+    "url": "discord-botlist.eu",
+    "description": {
+      "en": "Discord Bots is a cool Discord Bot List were users can add their Bots!",
+      "de": "Discord Bots ist eine coole Discord Bot-Liste, in der Benutzer ihre Bots hinzufügen können!"
```
Comment (Bas950) [inline websites/D/Discord BotList/dist/metadata.json:21]: please use an official description

File: websites/D/Discord BotList/presence.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  } else if (botPage == "/apply/certification/application") {
+    botData.details = "Applying for Certification";
+  } else if (botPage == "/tos/") {
+    botData.details = "Viewing ToS";
+    botData.buttons = [
+      {
+        "label": "View ToS",
+        "url": document.URL
+      }
+    ];
```
Comment (Bas950) [inline websites/D/Discord BotList/presence.ts:92]: dont think that one is needed.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/D/Discord BotList/presence.ts (hunk 2 above)
Comment (SuperzZyntexYT) [reply websites/D/Discord BotList/presence.ts]: @Bas950 fixed

File: websites/D/Discord BotList/presence.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+botPresence.on("UpdateData", async () => {
+  const botData: PresenceData = {
+    largeImageKey: "logo"
+  },
+    botPage = document.location.pathname,
+    botHost = document.location.hostname;
+
+  botData.startTimestamp = botBrowsing;
+
+  console.log(botHost)
```
Comment (Bas950) [inline websites/D/Discord BotList/presence.ts:15]: remove

File: websites/D/Discord BotList/presence.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    } else if (botPage == "/privacypolicy/") {
+      botData.details = "Viewing Privacy Policy";
+    } else if (botPage == "/Imprint/") {
+      botData.details = "Viewing Imprint";
+    }
+  } else if (botHost == "docs.discord-botlist.eu") {
+    const page = document.querySelector("#__GITBOOK__ROOT__CLIENT__ > div.reset-3c756112--body-68cac36c > div.reset-3c756112--bodyContent-2f98451b > div > div.reset-3c756112--wholeContentBody-554be184 > div.reset-3c756112--wholeContentPage-6c3f1fc5 > div > div.reset-3c756112--pageContainer-544d6e9c > div.reset-3c756112 > div.reset-3c756112--pageHeader-15724735 > div > div > div.reset-3c756112--horizontalFlex-5a0077e0 > div.reset-3c756112--pageHeaderIntro-0c1463da > h1 > span").textContent;
+    botData.details = "Viewing Docs";
+    botData.state = "Page: " + page;
+  }
```
Comment (Bas950) [inline websites/D/Discord BotList/presence.ts:100]: upload proof of working

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/D/Discord BotList/presence.ts (hunk 4 above)
Comment (OGMatrix) [reply websites/D/Discord BotList/presence.ts:98]: @Bas950 first done, second:
ss6
ss5

**Review by Bas950 — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]

**Pull request merged**